inductiva/utils/grids.py

```diff
@@ -0,0 +1,64 @@
+"""Utility functions to manipulate grid data."""
+import typing
+
+import scipy
+import numpy as np
+
+
+def get_meshgrid(x_range: typing.Sequence[float],
+                 y_range: typing.Sequence[float], num_list: typing.List[int]):
+    """Create a grid of x and y values.
```

Comment:
why do we need a num_list?
why not x_num e y_num?
Otherwise it would be fair to ask: why don't we have range_list?